cea/datamanagement/format_helper/format_helper.py

```
@@ -0,0 +1,99 @@
+"""
+Main script of the formate helper that activates the verification and migration as needed.
+
+"""
+
+import cea.config
+import os
+import subprocess
+import sys
+import cea.config
```

Ruff (0.8.2)

Redefinition of unused `cea` from line 6

Remove definition: `cea`

(F811)